packages/@webex/internal-plugin-metrics/src/call-diagnostic/types/ClientEvent.json

```diff
@@ -0,0 +1,5428 @@
+{
+  "$ref": "#/definitions/ClientEvent",
```

Comment:
does this cover all the call analyzer events ?

Reply:
Call Analyzer team defined multiple types of events in the https://sqbu-github.cisco.com/WebExSquared/event-dictionary repo.

the main categories are:
- ClientEvent (used to determine JMF, JMS, JMT, FFD, share results, call drops etc)
- MQE (Media Quality Events)
- FeatureUsageEvents (tracking feature usage on UCF, data that gets processed and vizualised in webex oneview.

All those types are defined in RAML file, but we managed to get them converted to typescript and JSON schema, so when we build the events we can assure the structure (types) and semantics (JSON schema validation).